src/styles/main.scss

```diff
@@ -1,3 +1,224 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+:root {
+  --background-color: #fff;
+  --main-accent-color: #060b35;
+  --blue-hover-color: #34568b;
+  --second-color: #616070;
+  --blue-accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0,0.1);
+  width: 100%;
```

Comment:
width 100% in header makes horizontal scroll and some tests failed
for fix it add such styles
```
* {
  box-sizing: border-box;
}
```